iron-menu-behavior.html

```diff
@@ -41,14 +41,20 @@
        */
       attrForItemTitle: {
         type: String
+      },
+
+      disabled: {
+        type: Boolean,
+        value: false,
+        reflectToAttribute: true,
```

Comment:
I looked into this. @danbeam had mentioned that code uses hasAttribute('disabled'), but I only saw that being used on the items, rather than the host. I've removed reflectToAttribute